lib/server/src/server.rs

```diff
@@ -573,11 +605,14 @@ impl TrowServer {
             let mut have_manifest = false;
             //Get auth token
 
-            let auth_token = if let Some(auth) = &proxy_auth {
-                self.get_auth_token(&cl, &proxy_image, &auth).await
-            } else {
-                None
+            let auth_token = match self.get_auth_token(&cl, &proxy_image, &proxy_auth).await {
+                Ok(a) => Some(a),
+                Err(e) => {
+                    error!("Can't get auth_token: {}", e);
```

Comment:
Let's expand this a bit so it makes more sense to the user e.g. "Failed to get auth token for IMAGE with error ERR"